.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: ActiveRecord MySQL Enum
+on: [push]
```

Comment:
Is the `pull_request` specification here actually unnecessary? I've seen it being used everywhere but all your commits were being tested 🤯

Reply:
`pull_request` is necessary if you want to make sure that your suite re-runs on pull request creation/updates/etc. If you just care that every commit is built against, then all you need is `push`